core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoEntityProvider.scala

```diff
@@ -34,10 +35,13 @@ class DataRepoEntityProvider(snapshotModel: SnapshotModel, requestArguments: Ent
   implicit val contextShift: ContextShift[IO] = IO.contextShift(executionContext)
   override val entityStoreId: Option[String] = Option(snapshotModel.getId)
 
-  private lazy val googleProject = {
-    // determine project to be billed for the BQ job TODO: need business logic from PO!
+  private[datarepo] lazy val googleProject: GoogleProjectId = {
```

Comment:
is there a reason to not universally favor `GoogleProject` over `GoogleProjectId` since they have the same footprint?

Reply:
I guess I prefer `GoogleProjectId` because it's part of the Rawls codebase instead of an imported library ... but I also didn't realize both exist. Hmm. Looks like `GoogleProjectId` is only 5 weeks old.

Reply:
also - `requestArguments.workspace.googleProject` is a `GoogleProjectId`, and that's the expected code path.